tooltip.css

```diff
@@ -0,0 +1,116 @@
+body {
+    font-family: sans-serif; 
+    display: flex; /* Используем flexbox для центрирования содержимого */
+    justify-content: center; /* Выравнивание содержимого по горизонтали */
+    align-items: center; /* Выравнивание содержимого по вертикали */
+    height: 50vh; 
+}
+  
+/* Стили для контейнера с границей + анимация */
+#border {
+    border: 2px solid rgb(18, 57, 1); 
+    margin: auto; /* Центрируем по горизонтали */
+    padding: 20px; /* Отступы внутри контейнера */
+    text-align: center; 
+    opacity: 0; /* Изначально контейнер скрыт */
+    animation: slideFromLeft 0.8s linear forwards; /* linear - движение элемента происходит с постоянной скоростью */
+    animation-delay: 1s; /* Задержка в 1 секунду */
+}
+
+@keyframes slideFromLeft {
+    from {
+        transform: translateX(-100%);
+        opacity: 0;
+    }
+    to {
+        transform: translateX(0);
+        opacity: 1;
+    }
+}
+
+/* Стили для каждого триггера с тултипом */
+.tooltip {
+    position: relative; /* Позиционируем относительно своего родителя */
+    display: inline-block; /* Блочный элемент с возможностью выравнивания по горизонтали */
+    margin: 20px; /* Отступы вокруг элемента */
+    cursor: pointer; 
+    border: 2px solid rgb(167, 0, 0); 
+}
+
+.tooltip .tooltipleft, .tooltip .tooltiptop, .tooltip .tooltiptbottom, .tooltip .tooltipright {
+    visibility: hidden; 
+    width: 120px; 
+    background-color: #555; 
+    color: #fff; 
+    text-align: center; 
+    border-radius: 5px; 
+    padding: 5px 0; /* Отступы внутри тултипа */
+    position: absolute; 
+    z-index: 1; /* Поверх всех остальных элементов */
+    opacity: 0; /* Изначально тултип невидимый */
+    transition: opacity 0.3s; /* Плавное изменение прозрачности */
+}
+
+.tooltip .tooltipleft::after, .tooltip .tooltiptop::after, .tooltip .tooltiptbottom::after, .tooltip .tooltipright::after {
```

Comment:
Same idea here. If you give all tooltips common class you could set these styles to ```. tooltip_content::after {}``` and that's it.